scss/civicrm/common/_tables.scss

```diff
@@ -95,14 +95,6 @@ table {
     &.label {
       vertical-align: middle !important;
     }
-
-    .crm-table-group-summary td {
-      background: $crm-dark-blue !important;
-
-      &:last-child {
-        text-align: left;
```

Comment:
we don't need this anymore?

Reply:
We have moved style related to summery tooltip to common and this is just duplication of it which is not longer required.